bokeh/command/subcommands/static.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import
+
+import logging
+log = logging.getLogger(__name__)
+
+from bokeh.server.server import Server
+from bokeh.util.string import nice_join
+
+from ..subcommand import Subcommand
+
+LOGLEVELS = ('debug', 'info', 'warning', 'error', 'critical')
+
+class Static(Subcommand):
+    ''' Subcommand to launch the Bokeh static server. '''
+
+    name = "static"
+
+    help = "Run a Bokeh static server (hosting bokehjs' static files)"
+
+    args = (
+        ('--port', dict(
+            metavar='PORT',
+            type=int,
+            help="Port to listen on",
+            default=None
+        )),
+
+        ('--address', dict(
+            metavar='ADDRESS',
+            type=str,
+            help="Address to listen on",
+            default=None,
+        )),
+
+        ('--log-level', dict(
+            metavar='LOG-LEVEL',
+            action  = 'store',
+            default = 'info',
+            choices = LOGLEVELS,
+            help    = "One of: %s" % nice_join(LOGLEVELS),
+        )),
+    )
```

Comment:
This should be codified in a test (inching towards locking down API and CLI specifications)